fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -280,13 +277,9 @@ public Response deleteObject() {
 
         try {
             resource().delete();
-
-            try {
-                session.save();
-            } catch (final RepositoryException e) {
-                throw new RepositoryRuntimeException(e);
+            if (!batchService.exists(session.getId(), getUserPrincipal())) {
```

Comment:
@acoburn , this comment may have slipped through the cracks:
https://github.com/fcrepo4/fcrepo4/pull/1123#discussion_r83494182